processor/src/main/java/org/mapstruct/ap/internal/util/MapperConfiguration.java

```diff
@@ -100,8 +100,8 @@ public String implementationPackage() {
     public List<TypeMirror> imports() {
         List<TypeMirror> imports = new ArrayList<>();
         imports.addAll( mapperPrism.imports() );
-        if ( mapperConfigPrism != null ) {
-            imports.addAll( mapperConfigPrism.imports() );
+        if ( mapperConfigPrism != null && mapperConfigPrism.imports() != null) {
```

Comment:
Are you sure that `mapperConfigPrism.imports()` can return `null`? We don't do a `null` check on line 102 which is the same logic.